daml-lf/validation/BUILD.bazel

```diff
@@ -51,6 +52,28 @@ da_scala_library(
     ],
 )
 
+da_scala_library(
+    name = "validation-upgrades-test-lib",
+    srcs =
+        glob(
+            ["src/test/**/UpgradesSpecUtil.scala"],
+        ) +
+        #  we depend on sources to avoid pushing a canton artifact to maven
+        [
+            "//canton:community_admin-api_proto_scala",
+        ],
```

Comment:
Consider depending directly on a built canton artifact, like community_base, over recompiling the source code. Daml-script only does this as it is distributed, your tests are not

Reply:
For some reason I couldn't get it to build without copying this in as a source - got a bunch of odd errors.

Reply:
I tried to get this to work - will ping in morning, but for now going to merge so as not to stop the train